Source/WebCore/editing/SpellChecker.cpp

```diff
@@ -152,8 +152,8 @@ bool SpellChecker::isCheckable(const SimpleRange& range) const
     }
     if (!foundRenderer)
         return false;
-    Ref node = range.start.container.get();
-    return !is<Element>(node) || downcast<Element>(node.get()).isSpellCheckingEnabled();
+    RefPtr element = dynamicDowncast<Element>(range.start.container.get());
```

Comment:
I didn't look into `isSpellCheckingEnabled` but I doubt we really need a smart pointer here?

Reply:
It does node traversal (loop over parentOrShadowHostElement) and attribute lookup. You added the smart pointer 4 months ago as part of https://github.com/WebKit/WebKit/commit/389241b67619b49fab63ca2d9abafb5c2a69358a so I didn't want to regress that. Please confirm you definitely want to see it gone.

Reply:
`isSpellCheckingEnabled` is not super trivial. Let's keep the smart pointer.